src/base/InternalBaseBlock.php

```diff
@@ -41,6 +44,16 @@
 
     private $_injectorObjects;
     
+    public function __construct(array $config = [])
```

Comment:
I don't think the cachable trait is needed in InternalBaseBlock

Reply:
There we get some conflicts with the $cacheEnabled and $cacheExpiration property of `CacheableTrait` and `InternalBaseBlock`. So I implemented `CacheableTrait` into InternalBaseBlock to resolve the conflicts.

Reply:
good point, i think i should change this in the callable trait anyhow, cause its should be private and is only a fallback value if nothing provided.

Reply:
@boehsermoe so i am sorry for sending you this extra mile with the Cacheable trait, so we can revert to your solution, i will change the cachable trait  and implement this after next release. Sorry for that!